halogen/vnd/error.py

```diff
@@ -0,0 +1,72 @@
+"""vnd.error support."""
+
+import halogen
+import six
+
+
+class Error(Exception):
+    """Base exception."""
+
+    def __init__(self, message, path=None, errors=None):
+        """Create an error.
+
+        :param message: Error message.
+        :param path: Optional JSON Pointer path.
+        :param errors: Optional nested errors.
```

Comment:
this parameter should be a non-nested list of errors, right? Or do we allow nesting?

Reply:
This list of errors is embedded (nested) to the main error.